freqtrade/rpc/rpc.py

```diff
@@ -135,9 +135,11 @@ def _rpc_trade_status(self) -> List[Dict[str, Any]]:
                 trade_dict = trade.to_json()
                 trade_dict.update(dict(
                     base_currency=self._freqtrade.config['stake_currency'],
-                    close_profit=fmt_close_profit,
+                    close_profit=trade.close_profit or None,
```

Comment:
```suggestion
                    close_profit=trade.close_profit,
```
(i.e. should be 0, not None, if trade.close_profit is zero)

same above for fmt_close_profit: there `if trade.close_profit else None` should be changed to `if trade.close_profit is not None else None` I guess